maldoca/ole/oss_utils.cc

```diff
@@ -438,6 +439,72 @@ bool BufferToUtf8::ConvertEncodingBufferToUTF8String(absl::string_view input,
     return true;
   }
   const char* input_ptr = input.data();
+
+#if defined(_WIN32)
+  int mb_size = 0;  // size of the multi-byte string
```

Comment:
how about we just say bytes instead of size?  size is ambiguous.  and in wc_size it's the number of wide chars.

Reply:
done